lib/modules/ens/index.js

```diff
@@ -23,12 +91,12 @@ class ENS {
     this.doSetENSProvider = this.namesConfig.provider === 'ens';
 
     this.addENSToEmbarkJS();
-    this.configureContracts();
     this.registerEvents();
   }
 
   registerEvents() {
     this.events.once("contracts:deploy:afterAll", this.setProviderAndRegisterDomains.bind(this));
+    this.events.once("web3Ready", this.configureContracts.bind(this));
```

Comment:
I'd expect `configureContracts` to be properly bound. Is `.bind(this)` necessary here?

Reply:
Nope, it has to be bound. I double check